base/cvd/cuttlefish/common/libs/utils/tee_logging.cpp

```diff
@@ -133,6 +133,18 @@ TeeLogger::TeeLogger(const std::vector<SeverityTarget>& destinations,
                      const std::string& prefix)
     : destinations_(destinations), prefix_(prefix) {}
 
+ScopedTeeLogger::ScopedTeeLogger(TeeLogger tee_logger)
```

Comment:
Can this accept a `android::base::LogFunction` for parity with `android::base::SetLogger`?

Reply:
From chat conversation:

The purpose of using a `TeeLogger` parameter is to imply different behavior from typical android logging.  The constructor forcibly sets the `LogSeverity` to `VERBOSE` for its lifespan, which is specific to the `tee_logging` implementation.  

Will keep it as is, at least for now.